google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GrpcSpannerRpc.java

```diff
@@ -495,6 +504,31 @@ String extractHeader(String resource) {
     return elements.get((int) index);
   }
 
+  /**
+   * This is a one time setup for grpcz pages. This adds all of the methods to the Tracing
+   * environment required to show a consistent set of methods relating to Cloud Bigtable on the
+   * grpcz page.  If HBase artifacts are present, this will add tracing metadata for HBase methods.
+   */
+  private static void setupTracingConfig() {
```

Comment:
Note that this is needed for gRPC version < 1.8.0 (if you change to gRPC 1.8.0 and regenerate the gRPC stubs using gRPC-proto-1.8.0) this is no longer needed. So add a message to remove this.